### PR TITLE
'--allow-multiple-windows' to `--disable-multiple-windows`

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

File: src/errors/test-run/templates.js [hunk 1]
```diff
@@ -375,4 +371,12 @@ export default {
...
         ${escapeHtml(err.errMsg)}
     `,
+
+    [TEST_RUN_ERRORS.multipleWindowsModeIsDisabledError]: err => `
+        You disabled multi window mode. Remove the "disable-multiple-windows" run option to use the "${err.methodName}" method.
+    `,
+
+    [TEST_RUN_ERRORS.multipleWindowsModeIsNotAvailableInRemoteBrowserError]: err => `
+        Multi window mode is available only in local browsers. Run the test locally to use the "${err.methodName}" method.
+    `,
```
Comment (AlexKamaev) [inline src/errors/test-run/templates.js:381]: @VasilyStrelyaev

Comment (VasilyStrelyaev) [reply src/errors/test-run/templates.js:381]:
```js

     [TEST_RUN_ERRORS.multipleWindowsModeIsDisabledError]: err => `
        Multi window mode is disabled. Remove the '--disable-multiple-windows' CLI flag or set the 'disableMultipleWindows' property to 'false' in the API to use the '${err.methodName}' method.
   `,

     [TEST_RUN_ERRORS.multipleWindowsModeIsNotAvailableInRemoteBrowserError]: err => `
        Multi window mode is supported in local browsers only. Run tests locally to use the '${err.methodName}' method.
    `,

Comment (VasilyStrelyaev) [reply src/errors/test-run/templates.js:381]: Checked with Dirk

Comment (AlexKamaev) [reply src/errors/test-run/templates.js:381]: We have `disableMultipleWindows` as option of the `run` method of `runner`

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Pull request merged**